sql/hive/pom.xml

```diff
@@ -233,7 +233,7 @@
         <artifactId>scalatest-maven-plugin</artifactId>
         <configuration>
           <!-- Specially disable assertions since some Hive tests fail them -->
-          <argLine>-da -Xmx4g -XX:ReservedCodeCacheSize=${CodeCacheSize} ${extraJavaTestArgs} -Dio.netty.tryReflectionSetAccessible=true</argLine>
```

Comment:
Perhaps it cannot be deleted here, just like `extraJavaTestArgs`, some test cases do not use `JavaModuleOptions` to supplement default parameters, same as `SparkBuild.scala`

Reply:
I just moved it into `extraJavaTestArgs`

Reply:
OK